datalad/support/archives.py

```diff
@@ -244,6 +242,8 @@ def compress_files(files, archive, path=None, overwrite=True):
                 patoolib.util.check_new_filename(archive)
             patoolib.util.check_archive_filelist(files)
             # Call protected one to avoid the checks on existence on unixified path
+            # MIH: How could this possibly work on Windows? Shouldn't this be a native path
+            # patoolib.util.PatoolError: Command `['C:\\Users\\mih\\Miniconda3\\Library\\usr\\bin\\gzip.EXE', '-v', '-c', '-9', '--', '"fi le.dat"', '>', '/C/Users/mih/AppData/Local/Temp/datalad_temp_1qotze3o.gz']' returned non-zero exit status 1
```

Comment:
Here is the immediate issue.

Reply:
I guess it is patool bug, file an issue there or submit a pr... I think I have some super powers there

Reply:
Why should it be a patool bug? We are passing it a butchered, completely non-standard unix-like path on windows, and it uses it verbatim in a windows cmdline call. I don't understand why we do it to begin with. I was hoping you could shed light on the necessity of unixify_path for the archive location itself.

Reply:
it seems it was introduced back in 2015 in a7cf9b2d7021b83c73e2bfc61ca7893e92f5123a with a commit message saying

BF: overcome tar issues on Windows by converting filenames to /drive/path

I guess I had some initial attempts to make it work in some flavor of Python for Windows, probably msys2 based, thus probably wanted some posixy paths.